Console/CreateUserCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
Ge had hier beter eerst gecommit en dan gemoved en gecommit. Nu kunnen we niet zien wa je gewijzigd hebt